lib/cookies.js

```diff
@@ -91,7 +91,7 @@ Cookies.prototype.set = function(name, value, opts) {
 
   if (opts && signed) {
     if (!this.keys) throw new Error('.keys required for signed cookies');
-    cookie.value = this.keys.sign(cookie.toString())
+    cookie.value = this.keys.sign(cookie.toString()).toString('base64')
```

Comment:
I was using this test as a reference https://github.com/crypto-utils/keygrip/blob/master/test/test.js#L40